storm-data-contracts/src/main/java/com/forter/contracts/BaseContractsBoltExecutor.java

```diff
@@ -165,6 +184,10 @@ protected TInput transformInput(Object contract) throws JsonProcessingException
         }
     }
 
+    protected Optional<? extends CacheDAO<TInput, TOutput>> prepareCacheDAO(Map stormConf, TopologyContext context) {
+        return Optional.absent();
```

Comment:
Instead of Optional, return a default implementation that does not caching... it will make your code much smoother.